internal/provider/resource_tfe_oauth_client_test.go

```diff
@@ -153,6 +187,7 @@ resource "tfe_oauth_client" "foobar" {
   http_url         = "https://github.com"
   oauth_token      = "%s"
   service_provider = "github"
+  organization_scoped = true
```

Comment:
@laurenolivia the CI tests that are failing right now are part of test files that I haven't modified. The reason they are failing is because the resource needs an organization_scoped attribute which is defined under beta flag in atlas. This attribute is optional. Locally, when I add line 190 to all the other failing tests they pass, but I would still have to skip all the tests which is a bad idea. Is there a way to get around this or do I have to remove the feature flag in atlas first?

Reply:
It sounds like the current options we've identified are either:
1. make the feature GA
2. use our `skipUnlessBeta()` helper

And neither are an option for your current workflow? Can you say more about why option #2 is a bad idea?

Reply:
It not just this test, I would have to skip all the test that are failing by the CI tests right now (there are several). If the cli team is ok with this then I can go ahead and add it but...in the past the team has told me that skipping tests can be risky with other code changes that are added in.

Reply:
Would it be possible to exercise the `organization_scoped` attribute only in its own set of tests?